include/seqan/sequence/string_set_base.h

```diff
@@ -1477,6 +1477,7 @@ return iter(me, length(me), tag);
  */
 
 template<typename TString, typename TSpec, typename TId>
+// [[deprecated("Use the subscript operator (operator[]) instead.")]]
```

Comment:
The documentation marks those functions (```*ById()```) as deprecated with the comment that the ID is no different than the position. When I tried to replace those functions I encountered several errors and it seems this is related to the dependent tight StringSet. So I commented those macros out too, since changes to the stringset will probably not happen until seqan3.